docs/monster-ui/charsRemaining().md

```diff
@@ -0,0 +1,46 @@
+title: charsRemaining()
+
+# monster.ui.charsRemaining()
+
+## Syntax
+```javascript
+monster.ui.charsRemaining($target, args);
+```
+
+### Parameters
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`$target` | Form element to be checked. | `jQuery` | | `true`
+`args` | Aguments to setup the helper | `Object` | | `true`
+
+### args
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`args.size` | The maxlength to be validated | `Integer` | | `true`
```

Comment:
```suggestion
`size` | The maxlength to be validated | `Integer` | | `true`
```
(from [example](https://raw.githubusercontent.com/2600hz/monster-ui/master/docs/monster-ui/monthpicker().md))

Also, as stated in my previous review, [there is not such thing as a pure integer type in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Data_types); your should use `Number` instead.